traitsui/testing/tester/default_registry.py

```diff
@@ -0,0 +1,26 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+
+import importlib
+
+from traits.etsconfig.api import ETSConfig
+
+from traitsui.testing.tester.registry import TargetRegistry
+from traitsui.tests._tools import is_null
+
+
+def get_default_registry():
```

Comment:
Could we have a docstring describing what it is please? (same for the other functions)
(I did not have very many docstring for the draft branch either but I have an excuse: It was draft 😂 )